src/control/server/storage/bdev.go

```diff
@@ -31,7 +31,10 @@ import (
 )
 
 // BdevPciAddrSep defines the separator used between PCI addresses in string lists.
-const BdevPciAddrSep = " "
+const (
+	BdevPciAddrSep = " "
+	NilBdevAddress = "<nil>"
```

Comment:
minor: It would probably be better to make this match the zero value of `hardware.PCIAddress{}`, e.g. `0000:00:00.0` so that the relationship is clear and also so that if the string value winds up being parsed by that package the result makes sense.